tutorials/02_installation.md

```diff
@@ -1,17 +1,17 @@
 \page installation Installation
 
 These instructions are for installing only Ignition Physics.
-If you're interested in using all the Ignition libraries, check out this [Ignition installation](https://ignitionrobotics.org/docs/latest/install).
+If you're interested in using all the Ignition libraries, check out this [Ignition installation](https://ignitionrobotics.org/docs/citadel/install).
 
-We recommend following the Binary Installation instructions to get up and running as quickly and painlessly as possible.
+We recommend following the [binary installation](#Binary-installation) instructions to get up and running as quickly and painlessly as possible.
```

Comment:
I think these links had been removed because now we have multiple sections for binary and source installation, according to the OS.